iiasa/message_ix
# Adjust calculation of PRICE_EMISSION

This PR revises the calculation of the `PRICE_EMISSION` to correctly account for the period-duration when deriving the period-specific `PRICE_EMISSION` from a single marginal value which covers multiple periods (cumulative constraint).

## How to review

- Read the diff and note that the CI checks all pass.
- Build the documentation and look at revised documentation section.
- Ensure that changes/additions are self-documenting, i.e. that another
  developer (someone like the reviewer) will be able to understand what the code
  does in the future.

## PR checklist
